src/ui/lang.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+import { langArr } from "./translation.js";
+const langSelect = document.querySelector(".change-lang");
+const languages = ["ru", "en", "uz"];
+langSelect.addEventListener("change", changeURLLanguage);
+export function changeURLLanguage() {
+  let lang = langSelect.value;
+  location.href = window.location.pathname + "#" + lang;
+  location.reload();
```

Comment:
I think it's better to replace `location.reload()` with `changeLanguage()` here. This way we don't need to reload the page. Generally, manual reload of the page should be avoided (might lead to weird unexpected bugs), unless there are no other options.

Reply:
I tried to fix bugs. Please, test it one more time